site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -44,14 +47,21 @@ export const Language = {
   timezoneLabel: "Timezone",
   ttlLabel: "Time until shutdown (hours)",
   ttlHelperText: "Your workspace will automatically shut down after this amount of time has elapsed.",
+  ttlCausesShutdownHelperText: "Your workspace will shut down",
+  ttlCausesShutdownAt: "at",
+  ttlCausesShutdownImmediately: "immediately!",
+  ttlCausesShutdownSoon: "within 30 minutes.",
+  ttlCausesNoShutdownHelperText: "Your workspace will not automatically shut down.",
 }
 
 export interface WorkspaceScheduleFormProps {
   fieldErrors?: FieldErrors
   initialValues?: WorkspaceScheduleFormValues
   isLoading: boolean
+  now: dayjs.Dayjs
```

Comment:
*review:* I'm not sure if there's a better way to do it, but I figured it's easiest to just pass in the current time as a prop.

Reply:
**Comment(approving)**: I'm OK with this, it seems like we either don't do this and wrestle with mocks or spies in tests and storybook...or we just do this and live with it being a little weird.

Reply:
**Suggestion(nit)**: Just let's make it optional:


```suggestion
  now?: dayjs.Dayjs
```